### PR TITLE
Only count non-converted beatmaps

### PR DESCRIPTION
**Pull request by notbakaneko**

### CODE REVIEW

File: resources/assets/coffee/react/_components/playmode-tabs.coffee [hunk 1]
```diff
@@ -40,7 +40,10 @@ class @PlaymodeTabs extends React.Component
             'data-disabled': disabled
             osu.trans "beatmaps.mode.#{mode}"
             if @props.showCounts
-              count = _.size(@props.beatmaps[mode])
+              filtered = _.filter @props.beatmaps[mode], (beatmap) ->
```
Comment (nanaya) [inline resources/assets/coffee/react/_components/playmode-tabs.coffee]: you mean `_.reduce`

Comment (notbakaneko) [reply resources/assets/coffee/react/_components/playmode-tabs.coffee]: how about `no`

Comment (nanaya) [reply resources/assets/coffee/react/_components/playmode-tabs.coffee]: how about `sumBy`

Comment (notbakaneko) [reply resources/assets/coffee/react/_components/playmode-tabs.coffee]: `yes`

**Pull request merged**